pyproject.toml

```diff
@@ -38,13 +38,13 @@ opentelemetry-exporter-otlp-proto-http = "1.21.0"
 optional = true
 
 [tool.poetry.group.format.dependencies]
-ruff = "^0.4.5"
+ruff = "^0.12.7"
```

Comment:
Bump to latest ruff

Reply:
It's in PR https://github.com/canonical/mysql-operator/actions/runs/16738937383/job/47383262652?pr=425